src/pybind/mgr/dashboard/plugins/lru_cache.py

```diff
@@ -19,7 +19,7 @@ def decorating_function(function):
         stats = [0, 0]
         rlock = RLock()
         setattr(function, 'cache_info', lambda:
-                "hits={}, misses={}, maxsize={}, currsize={}".format(
+                "hits={}, misses={}, maxsize={}, cursize={}".format(
```

Comment:
This is intentional to comply with Python functools API:

https://github.com/python/cpython/blob/1c0a9c5a1c84bc334f2bde9d45676f19d9632823/Lib/functools.py#L430